### ISSUE TITLE
Upgrade macOS and Windows hosts for Eloquent patch release

### ISSUE THREAD

**Issue by nuclearsandwich** [ros2/build_farmer#259]
Continuation of https://github.com/ros2/build_cop/issues/243

The goal is to get the hosts upgraded close enough to the Eloquent release that the binaries have some chance of being valid before homebrew rolls out from under us but also to leave enough time to react to changes necessary in our code. In other words, I will endeavor to get to this tonight as soon as one of the mac hosts is free.

**Comment by nuclearsandwich** [comment]
Poco 1.10.0 became available via Homebrew but has an issue with the CMake configuration which declares it as 1.0.0. Since our vendor package requires at least 1.6.1 this triggers the vendor build in all cases. The issue is fixed in the devel branch and is listed in the 1.10.1 milestone (due Feb 17) https://github.com/pocoproject/poco/issues/2923

**Comment by wjwwood** [comment]
Just adding to this, warnings like this:

https://ci.ros2.org/view/nightly/job/nightly_osx_debug/1615/cmake/new/

Which had to be hacked around on Linux due to a newer assimp (5.0.1) which fixes the warning not being available (see: https://github.com/ros2/rviz/pull/534), could be fixed on macOS by upgrading to 5.0.1 via homebrew:

```
% brew info assimp
assimp: stable 5.0.1 (bottled), HEAD
Portable library for importing many well-known 3D model formats
https://www.assimp.org/
/usr/local/Cellar/assimp/5.0.1 (100 files, 12.3MB) *
  Poured from bottle on 2020-05-05 at 10:34:20
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/assimp.rb
==> Dependencies
Build: cmake ✔
==> Options
--HEAD
	Install HEAD version
==> Analytics
install: 1,232 (30 days), 3,916 (90 days), 13,867 (365 days)
install-on-request: 515 (30 days), 1,662 (90 days), 6,316 (365 days)
build-error: 0 (30 days)
```

**Comment by clalancette** [comment]
We aren't using this repository anymore for buildfarm issues, so I'm going to archive it.  Thus I'm closing out this issue.  If you continue to have problems, please report another bug against https://github.com/ros2/ros2.  Thank you.
